### PR TITLE
 [1LP][RFR] Fixed a few more parametrizations

### PR DESCRIPTION
**Pull request by psav**
A few more parametrization optimizations

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit a3795669e7e4e9d5eeb007f4c6917e03a7d1df26:

`cfme/services/myservice.py`:
- :heavy_exclamation_mark: Line 152:1: **W293** *blank line contains whitespace*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/webui/test_splitter.py [hunk 1]
```diff
@@ -35,10 +35,9 @@
 
 pytestmark = [
     pytest.mark.parametrize(
-        "location", LOCATIONS, ids=[
-            "{}-{}".format(loc[0].__name__, loc[1]) for loc in LOCATIONS]
+        "model_object,destination", LOCATIONS
```
Comment (izapolsk) [inline cfme/tests/webui/test_splitter.py]: :1st_place_medal:

File: cfme/tests/webui/test_splitter.py [hunk 2]
```diff
@@ -51,16 +50,16 @@
 )
```
Comment (izapolsk) [inline cfme/tests/webui/test_splitter.py]: REQUIRED: you forgot to change location above

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/webui/test_splitter.py (hunk 2 above)
Comment (psav) [reply cfme/tests/webui/test_splitter.py]: I'm not sure I follow?

Comment (mshriver) [reply cfme/tests/webui/test_splitter.py]: psav on L43 in your branch, there's a use of location within `unblock=lambda location: ...`

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/webui/test_splitter.py (hunk 2 above)
Comment (psav) [reply cfme/tests/webui/test_splitter.py]: not any more

**Pull request merged**